scripts/pi-hole/js/index.js

```diff
@@ -1210,3 +1209,8 @@ $(function () {
 window.addEventListener("resize", function () {
   $(".chartjs-tooltip").remove();
 });
+
+// change sidebar icon on collapse/expand
+$(".sidebar-toggle-svg").click(function () {
+  $(this).find("svg").toggleClass("fa-angle-double-left fa-angle-double-right");
+});
```

Comment:
There's no need to use javascript.

You can achieve the same effect using only CSS (in `pi-hole.css`):

```css
.sidebar-collapse .sidebar-toggle-svg {
  transform: scaleX(-1);
}

@media (max-width: 767px) {
  .sidebar-toggle-svg {
    transform: scaleX(-1);
  }
  .sidebar-open .sidebar-toggle-svg {
    transform: scaleX(1);
  }
}
```

Reply:
Done